modules/consul/exporter.tf

```diff
@@ -16,6 +16,123 @@ resource "helm_release" "consul_exporter" {
   ]
 }
 
+locals {
+  exporter_volume = "consul-ca"
+
+  exporter_server_ca_init = {
+    name  = "server-ca"
+    image = "alpine"
+    volumeMounts = [
+      {
+        name      = local.exporter_volume
+        mountPath = "/${local.exporter_volume}"
+      },
+    ]
+    resources = {
+      requests = {
+        cpu    = "50m"
+        memory = "25Mi"
+      }
+      limits = {
+        cpu    = "50m"
```

Comment:
I hope this is sufficient since hardcoding the values

Reply:
Should be, for now.